server/mediaserver_test.go

```diff
@@ -586,10 +586,6 @@ func TestCreateRTMPStreamHandler(t *testing.T) {
 	if err := handler(u, st); err != nil {
 		t.Error("Handler failed ", err)
 	}
-	// Test collisions via stream reuse
-	if sid := createSid(u); sid != nil {
```

Comment:
Instead of removing this test completely should we modify it to check for the absence of a failure when there is a naming collision?

Reply:
Makes sense, changed it while rebase+autosquash!